### PR TITLE
[UPD] website_baidu_map

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

**Comment by elicoidal** [comment]
Shouldnot this one target the OCA/l10n-china?
@noahzaozao

File: website_baidu_map/README.rst [hunk 1]
```diff
@@ -1,12 +1,10 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-   :alt: License: AGPL-3
+    :alt: License
 
-=================
 website_baidu_map
-=================
+================================
```
Comment (noahzaozao) [inline website_baidu_map/README.rst]: too much ===

File: website_baidu_map/__init__.py [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_baidu_map/__init__.py]: change the header

File: website_baidu_map/__openerp__.py [hunk 3]
```diff
@@ -1,20 +1,34 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_baidu_map/__openerp__.py]: change the header

File: website_baidu_map/__openerp__.py [hunk 4]
```diff
@@ -1,20 +1,34 @@
...
+{'name': 'Website Baidu Map',
+ 'version': '0.1',
```
Comment (noahzaozao) [inline website_baidu_map/__openerp__.py]: 'version': '8.0.1.0.1',

**Comment by noahzaozao** [comment]
LGTM
@elicoidal @seb-elico  Please review this PR , thanks.

File: website_baidu_map/README.rst [hunk 5]
```diff
@@ -27,12 +45,14 @@ Contributors
 Maintainer
 ----------
 
-.. image:: https://www.elico-corp.com/logo.png
-:alt: Elico Corp
-:target: https://www.elico-corp.com
+.. image:: http://odoo-community.org/logo.png
```
Comment (elicoidal) [inline website_baidu_map/README.rst]: Elico Footer

**Comment by elicoidal** [comment]
Check the README. Other than that LGTM

**Comment by noahzaozao** [comment]
LGTM FT

**Comment by seb-elico** [comment]
@noahzaozao "LGTM FT" is for Sita repo only. For normal repos, rule is: the reviewer who writes the second LGTM merges at the same time (if there was no comment between the first LGTM and the second LGTM)

**Pull request merged**